routes/tweet.js

```diff
@@ -1,32 +1,270 @@
 const router = require("express").Router();
 let Tweet = require("../models/tweet.model");
+let User = require("../models/user.model");
+var ObjectID = require('mongodb').ObjectID;
 
-router.route("/").get((req, res) => {
-  Tweet.find()
+router.route("/getUserTweets").get((req, res) => {
+
+
+
+  Tweet.find(req.query).sort('-createdAt')
+    .then((tweets) => res.json(tweets))
+    .catch((err) => res.status(400).json("Error: " + err));
+});
+
+router.route("/getTweet").get((req, res) => {
+
+  Tweet.find({ _id: req.query.tweetUUID })
```

Comment:
I wouldnt return a JSON object with tweets plural because this only returns a single tweet? If im wrong then disregard but thats what it looks like this route does